carbonserver/tests/api/service/test_sign_up_service.py

```diff
@@ -71,53 +47,35 @@
     name="Gontran Bonheur",
     email="xyz@email.com",
     api_key=API_KEY,
-    teams=[],
-    organizations=[ORG_ID],
```

Comment:
you remove the organization from the user ?

Reply:
Not really! I am removing the `USER_IN_DEFAULT_ORG_AND_TEAM` (not used anymore), it is just the `git diff` that is showing it weird.